### ISSUE TITLE
Multiple web console bugs

### ISSUE THREAD

**Issue by ryzhyk** [feldera/feldera#556]
- The throughput graph seems to always show 0 throughput now (I am testing it with the secops demo).
- The view and insert icons no longer work for me
- When I open the pipelines view, some of the idle pipelines appear expanded and there is no chevron icon to collapse them.

I see this behavior on both Firefox and Chrome.

**Comment by Karakatiza666** [comment]
> The throughput graph seems to always show 0 throughput
Can't reproduce on the latest main build - and never encountered this. Maybe demo itself was misconfigured?
> The view and insert icons
Are you referring to these ones?
![image](https://github.com/feldera/feldera/assets/8537259/37934fad-25ea-4a1c-ac38-9281c1ab767d)

**Comment by lalithsuresh** [comment]
@Karakatiza666 I encountered it when running the Docker demo yesterday:

```
curl https://raw.githubusercontent.com/feldera/dbsp/main/deploy/docker-compose.yml | docker compose -f - --profile demo up --force-recreate -V --pull always
```

**Comment by gz** [comment]
>The view and insert icons no longer work for me
this is/was likely related to the 100% cpu util issue (that the buttosn no longer work)

**Comment by lalithsuresh** [comment]
I can also add that when I clicked the button, the loading bar at the top was crawling slowly (more evidence for the CPU issue being the cause). I had to refresh the browser for it to actually move to that page correctly.

**Comment by Karakatiza666** [comment]
Fixed with d76c1360e4635582a944481166f0ea853c08a9f6, 60e7ddc3dffe44b0113273db0b31171e12692bd4, 6919692284aba2ae763c738497b5a00aa8489fa6
